### ISSUE TITLE
more control over lobby view camera

### ISSUE THREAD

**Issue by blairmacintyre** [mozilla/hubs-cloud#25]
Would like it if it was possible to disable the slow pan on the lobby view, or control it somehow.  If the lobby will be used for viewing a room (say, for a large group viewing a talk), we'd like the view to be in a defined good place, and not have it pan around.

**Comment by gfodor** [comment]
I think we should consider adding the ability for a room owner to enable lobby locomotion (basically, letting people in the lobby fly around the room, etc) Off by default but makes sense for trusted meetup contexts. Would that solve this problem?

**Comment by blairmacintyre** [comment]
Who would have control?  Would the room owner pick someone in the lobby?

**Comment by gfodor** [comment]
no, what i mean is that if this setting were enabled for a room, when you loaded the page, you could use WASD (and perhaps other controls) to fly around in the lobby.

**Comment by gfodor** [comment]
also, we currently have "Camera Mode" if you haven't discovered it yet, which likely doesn't solve your problem but may provide some value.

**Comment by gfodor** [comment]
(in camera mode, a room owner/moderator can control the camera that is seen from the lobby)

**Comment by blairmacintyre** [comment]
When people fly around in the lobby, can they hear from their viewpoint, even though people can't see them?

**Comment by gfodor** [comment]
correct
